src/core/operation.c

```diff
@@ -54,11 +54,11 @@ QuicOperationQueueUninitialize(
 _IRQL_requires_max_(DISPATCH_LEVEL)
 QUIC_OPERATION*
 QuicOperationAlloc(
-    _In_ QUIC_WORKER* Worker,
     _In_ QUIC_OPERATION_TYPE Type
     )
 {
-    QUIC_OPERATION* Oper = (QUIC_OPERATION*)CxPlatPoolAlloc(&Worker->OperPool);
+    QUIC_OPERATION* Oper =
+        (QUIC_OPERATION*)CxPlatPoolAlloc(&QuicLibraryGetPerProc()->OperPool);
```

Comment:
Isn't this going to slow down every allocation because it'll be a function call (to get the current processor), an array indexing operation, and then a pointer dereference, just to hand the pool off to the `CxPlatPoolAlloc` call?

Reply:
Not really, and besides, this is very temporary. The very next change (already mostly complete) is to move all these to use an explicit `Partition->OperPool` access,